cognitiveservicesPlugin/tests/tests.js

```diff
@@ -0,0 +1,7 @@
+exports.defineAutoTests = function () {
+    describe('Plugin tests', function() {
+        it('test something', function() {
```

Comment:
Should we add a real test here, rather than a sample?

Reply:
We could, I was thinking we should just get the framework in place, then start adding tests